.travis.yml

```diff
@@ -0,0 +1,85 @@
+sudo: required
+dist: trusty
+language: cpp
+compiler:
+    # FIXME: For now, building with Clang is disabled because the STP built with
+    # it hits an assertion failure during some tests. We should sort this out
+    # eventually and file the bug against STP or Clang whichever is appropriate,
+    # but for now it is easier to just reduce the number of configs we test
+    # anyway.
+    # - clang
+
+    - gcc
+env:
+    ###########################################################################
+    # Configurations
+    #
+    # Each line in the "env" section represents a set of environment variables
+    # passed to a build. Thus each line represents a different build
+    # configuration.
+    ###########################################################################
+
+    # Check a subset of the matrix of:
+    #   LLVM  : {2.9, 3.4}
+    #   SOLVERS : {STP:Z3}
+    #   STP_VERSION : {2.1.2}
+    #   DISABLE_ASSERTIONS: {0}
+    #   ENABLE_OPTIMIZED: {1}
+    #   USE_CMAKE: {0, 1}
+
+    matrix:
+    # Check KLEE CMake build in a few configurations
+    - LLVM_VERSION=3.4 SOLVERS=STP:Z3 STP_VERSION=2.1.2 DISABLE_ASSERTIONS=0 ENABLE_OPTIMIZED=1 USE_CMAKE=1
```

Comment:
Drop STP and just use Z3. Our only concern real concern is klee-uclibc and STP has to be built from source. For Z3 we use a package so the build is faster.